tests/app_settings.rs

```diff
@@ -9,7 +9,8 @@ USAGE:
 
 FLAGS:
     -h, --help       Print help information
-    -V, --version    Print version information";
+    -V, --version    Print version information
+";
```

Comment:
I can guess the motivation but could you list it out explicitly?

Reply:
The motivation was specifically https://github.com/clap-rs/clap/issues/2787 where the help message does not include a trailing newline.

This change also highlights areas where there are multiple trailing newlines in the help message output

Reply:
Thanks!